rotkehlchen/db/dbhandler.py

```diff
@@ -3421,3 +3421,25 @@ def delete_user_note(self, identifier: int) -> None:
             write_cursor.execute('DELETE FROM user_notes WHERE identifier=?', (identifier,))
             if write_cursor.rowcount == 0:
                 raise InputError(f'User note with identifier {identifier} not found in database')
+
+    def get_nft_mappings(self, identifiers: List[str]) -> Dict[str, dict]:
+        """
+        Given a list of nft identifiers, return a list of nft info (id, name, collection_name)
+        for those identifiers.
+        """
+        result = {}
+        identifiers_query = f'identifier IN ({",".join("?" * len(identifiers))})'
+        with self.conn.read_ctx() as cursor:
+            cursor.execute(
+                'SELECT identifier, name, collection_name FROM nfts WHERE ' + identifiers_query,
+                identifiers,
+            )
+            for entry in cursor:
+                result[entry[0]] = {
+                    'name': entry[1],
+                    'symbol': None,  # for compliance with assets mappings from globaldb
+                    'asset_type': 'nft',
```

Comment:
```suggestion
                    'asset_type': AssetType.NFT,
```

but I'm seeing that we don't return the asset type in `get_assets_mappings`. Maybe we should add it also there so the schema is the same for all the assets?

Reply:
If I understand correctly about which place you are talking, then in this PR I have changed it from `is_custom_asset` to `asset_type`, so it should be returned now. Or are you talking about something different?
![image](https://user-images.githubusercontent.com/63492346/200548970-dc5ef864-b9b2-480b-a868-15ad4d6adc52.png)

Reply:
This is not resolved. You still have `nft` hardcoded